R/detect_entities.R

```diff
@@ -25,7 +26,11 @@ function(
         out <- comprehendHTTP(action = "BatchDetectEntities", body = bod, ...)
         # build response data frame
         x <- out$ResultList
-        x <- cbind(Index = x$Index, do.call("rbind", x$Entities))
+        x <- purrr::map2(x$Index, x$Entities,
```

Comment:
If purrr is only for this map2() function could it be replaced with base `mapply()` ?  Will mean one less dependency